R/error-checks.R

```diff
@@ -136,11 +148,13 @@ hasUniqueValues <- function(data, na.rm = TRUE) {
 
 typeNamesFrom <- function(type) {
```

Comment:
Should these functions be hidden? If yes, do not export them (#internal) and use a dot at the beginning of function name. Otherwise, some description should be added?